tests/test_model.py

```diff
@@ -18,435 +18,461 @@
     jacobian_from_model, hessian_from_model, ModelError
 )
 
-class TestModel(unittest.TestCase):
-    """
-    Tests for Model objects.
-    """
-    def test_model_as_dict(self):
-        x, y_1, y_2 = variables('x, y_1, y_2')
-        a, b = parameters('a, b')
 
-        model_dict = OrderedDict([(y_1, a * x**2), (y_2, 2 * x * b)])
-        model = Model(model_dict)
+"""
+Tests for Model objects.
+"""
 
-        self.assertEqual(id(model[y_1]), id(model_dict[y_1]))
-        self.assertEqual(id(model[y_2]), id(model_dict[y_2]))
-        self.assertEqual(len(model), len(model_dict))
-        self.assertEqual(model.items(), model_dict.items())
-        self.assertEqual(model.keys(), model_dict.keys())
-        self.assertEqual(list(model.values()), list(model_dict.values()))
-        self.assertTrue(y_1 in model)
-        self.assertFalse(model[y_1] in model)
-
-    def test_order(self):
-        """
-        The model has to behave like an OrderedDict. This is of the utmost importance!
-        """
-        x, y_1, y_2 = variables('x, y_1, y_2')
-        a, b = parameters('a, b')
-
-        model_dict = {y_2: a * x**2, y_1: 2 * x * b}
-        model = Model(model_dict)
 
-        self.assertEqual(model.dependent_vars, list(model.keys()))
+def test_model_as_dict():
+    x, y_1, y_2 = variables('x, y_1, y_2')
+    a, b = parameters('a, b')
 
+    model_dict = OrderedDict([(y_1, a * x**2), (y_2, 2 * x * b)])
+    model = Model(model_dict)
 
-    def test_neg(self):
-        """
-        Test negation of all model types
-        """
-        x, y_1, y_2 = variables('x, y_1, y_2')
-        a, b = parameters('a, b')
+    assert id(model[y_1]) == id(model_dict[y_1])
+    assert id(model[y_2]) == id(model_dict[y_2])
```

Comment:
`model[y_1] is model_dict[y_1]`. But can be left for later

Reply:
It's not a big thing, right? I changed it :D

Reply:
Awesome :)